docs/equiv-check/index.md

```diff
@@ -0,0 +1,133 @@
+(equivcheck-intro)=
+Equivalence Checking Using the Certora Prover
+=================================
+
+This chapter describes how one can use the Certora Prover to
+  check the equivalence of two pure functions.
+
+```{note}
+We are working on extending this to support functions with side effects;
+  for the remaining documentation,
+  we assume a user is interested in running the equivalence checker
+  for two pure functions which may belong to the same contract
+  or two different contracts.
+```
+
+The equivalence checker front-end automatically generates (1) a
+  CVL spec to check if two functions are equivalent, and, (2) a
+  configuration file (`.conf`) for running the Certora Prover.
+
+## Installation
+
+The front-end script, `certoraEqCheck`,
+  should be available as part installing `certora-cli`.
+
+
+(example)=
+## Example
+
+Consider two contracts, `BasicMathGood.sol` and `BasicMathBad.sol` shown
+  below with two functions, `add` and `add_mult`.
+
+```
+BasicMathGood.sol:
+------------------
+
+contract BasicMathGood {
+    function add(uint256 a, uint256 b) public pure returns(uint256) {
+        return a + b;
+    }
+}
+
+```
+
+```
+BasicMathBad.sol:
+------------------
+
+contract BasicMathBad {
+    function add_mult(uint256 a, uint256 b) public pure returns(uint256) {
+        return a * b;
+    }
+}
+```
+
+We are interested in checking the equivalence of `add` and `add_mult`.
```

Comment:
might be nice to do an example with an `unchecked` block, or maybe even a line or two of assembly to sell it.  Then the functions won't "clearly" be non-equivalent.